Library/Tracking/AttributionTracking.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+struct AttributionTracking {
+  public enum AttributionEvent: String {
+    case projectPageViewed = "Project Page Viewed"
+  }
+
+  static func eventParametersString(refInfo: RefInfo?) -> String? {
+    let sessionRefTag = refInfo?.refTag?.stringTag
+    let contextPageUrl = refInfo?.deeplinkUrl
+    var props = [String: String]()
+    if let sessionRefTag {
+      props["session_ref_tag"] = sessionRefTag
+    }
+    props["context_page_url"] = contextPageUrl ?? ""
+
+    let propsData = try? JSONSerialization.data(withJSONObject: props)
+    return String(data: propsData!, encoding: .utf8)
```

Comment:
Is the force unwrap here necessary? Looks like you're already returning `String?` so I assume a `nil` would be harmless.

Reply:
Good catch! I like to force unwrap things when I'm testing but I didn't mean to leave it in. Fixed.